dynamic_rest/serializers.py

```diff
@@ -510,7 +510,7 @@ class WithDynamicModelSerializerMixin(WithDynamicSerializerMixin):
 
     @classmethod
     def get_model(cls):
-        return cls.Meta.model
+        return getattr(cls.Meta, 'model', None)
```

Comment:
No, because it will be automatically injected. See https://github.com/AltSchool/dynamic-rest/blob/master/dynamic_rest/serializers.py#L133-L136